examples/healthchecks/README.md

```diff
@@ -82,6 +86,10 @@ and for example here is a filled in database where:
 which when filled out, will look like this
 
 ```python
+INSTALLED_APPS = [
```

Comment:
When I suggested to remove the dots, I thought you would fill in apps that are needed. `django_spanner` isn't going to be the only app in INSTALLED_APPS. You need to copy all the other apps from the INSTALLED_APPS in hc/settings.py. (Same idea applies to the other examples.)

Reply:
The problem with that is that all the apps have to be listed, the goal is to just show what needs to be added to the front.

Reply:
I suggest to at least put the three dots back in the first mention of INSTALLED_APPS and remove INTSALLED_APPS from the "here is a filled in database" section (it seems to only distract from the purpose of that).

Reply:
Ack, thanks!